src/main/scala/com/gu/release/ShipAutoPlugin.scala

```diff
@@ -29,15 +27,41 @@ object ShipAutoPlugin extends AutoPlugin {
 
   lazy val buildingNewVersion = settingKey[Boolean]("whether we're building a new version to ship")
 
-  override def projectSettings: Seq[Def.Setting[_]] = ChangeLogBuild.settings ++
+  val shipToSonatype = Command.command("shipToSonatype") { state =>
+    "test" ::
+      "publishSigned" ::
+      "sonatypeReleaseAll" ::
+      state
+  }
+
+  /**
+    * Convert the given command string to a release step action, preserving and invoking remaining commands
+    * stolen from https://github.com/sbt/sbt-release/blob/7e0abd698f66f75f9e85ebd0f722b67b581d96c3/src/main/scala/ReleasePlugin.scala#L102
+    */
+  def releaseStepCommandAndRemaining(command: String, log: Logger): State => State = { st: State =>
```

Comment:
I would avoid using this code if possible but we can leave it like this for now if it works